### PR TITLE
Migrate optparse to argparse.

### PR DESCRIPTION
**Pull request by alexanderrichards**

### CODE REVIEW

**Comment by amaltaro** [comment]
@alexanderrichards from the meeting, I understood you're still going to add 4 more files to this PR, isn't it?

**Comment by alexanderrichards** [comment]
yes that's right its not finished yet

File: test/data/Alerts/alerts-load_example_data.py [hunk 1]
```diff
@@ -11,21 +11,20 @@
...
-from optparse import OptionParser, TitledHelpFormatter
+from argparse import ArgumentParser
 
 from WMCore.Database.CMSCouch import CouchServer, Database, Document
 
 
 def _processCmdLineArgs(args):
     usage = \
 """usage: %prog options"""
-    form = TitledHelpFormatter(width = 78)
```
Comment (alexanderrichards) [inline test/data/Alerts/alerts-load_example_data.py]: Note I didn't find an `argparse` equivalent of `TitledHelpFormatter`. Do we care about this?

**Review by alexanderrichards — COMMENTED** [review]
OK all done I think (of course tests will tell 😉) barring the one question below.

**Comment by ticoann** [comment]
test this please

**Comment by alexanderrichards** [comment]
test this please

**Comment by amaltaro** [comment]
Superseeded by #8849

**Pull request closed without merging**